### PR TITLE
Summary in profile page

### PR DESCRIPTION
**Pull request by davilaerte**
### Added the following information in the profile page:

- Description
- Location
- Email
- Organizations 
 
_task 02.1 of sprint 2_

### CODE REVIEW

**Comment by liraop** [comment]
Nice PR, specially the organizations view holder.

**Comment by liraop** [comment]
Nice PR, specially the organizations view holder.

File: app/src/main/java/com/github/ghcli/ViewHolder/OrganizationsViewHolder.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import android.widget.TextView;
+
+import com.github.ghcli.R;
+import com.squareup.picasso.Picasso;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by davi on 18/01/2018.
```
Comment (GabrielSVinha) [inline app/src/main/java/com/github/ghcli/ViewHolder/OrganizationsViewHolder.java]: I believe this was auto-generated. Can you remove it please?

Comment (davilaerte) [reply app/src/main/java/com/github/ghcli/ViewHolder/OrganizationsViewHolder.java]: Okay, done.

**Pull request merged**